### DISCUSSION TITLE
Is there a way to create navigation using API?

### DISCUSSION THREAD

**Discussion by vladimir-djokic-quantox** [VirtusLab/strapi-plugin-navigation#415] [General]
Is it possible to create navigation using API (anything would work for me - GQL, REST, Service API...)?

**Comment by CodeVoyager** [comment]
Hi @vladimir-djokic-quantox,

At the moment creating/mutating endpoints are available for admin endpoints only. To learn more you can consult source of this package or check network traffic in the admin panel.

**Reply by vladimir-djokic-quantox** [reply]
Thank for your response, I have managed to use API to create navigation:

```
GET    /navigation                   -> gets all navigations
POST   /navigation                   -> creates new navigations. You can pass items right away, but they will be for en locale only
DELETE /navigation/[NAVIGATION_ID]   -> deletes navigation
PUT    /navigation/[NAVIGATION_ID]   -> updates existing navigation. This is great for creating items that are for non-en locale
```

Don't forget to pass for APIs that require it

```
Content-Type: application/json
Authorization: Bearer YOUR_TOKEN
```
